src/main/java/seedu/duke/ui/TextUi.java

```diff
@@ -82,54 +84,60 @@ private void printList(List<Person> persons) {
      */
     private void printCheckedInList(List<Person> persons) {
         assert persons != null : "Does not print null";
+        printDivider();
         out.printf((PRINT_LIST_FORMAT) + "%n", " ", "Name", "Id", "Phone");
         printDivider();
         printPersonsInListWithoutStatus(persons);
+        printDivider();
     }
 
     private void printPersonsInListWithoutStatus(List<Person> persons) {
         for (int i = 0; i < persons.size(); ++i) {
             Person peronSelected = persons.get(i);
             String name = peronSelected.getName().getNameString();
+            String truncatedName = nameTruncator(name);
             String idString = peronSelected.getId().getIdString();
             String phoneString = (peronSelected.getPhone().isAvailable())
-                    ? "--" : peronSelected.getPhone().getPhoneNo();
-            out.printf((PRINT_LIST_FORMAT) + "%n", i + 1, name, idString, phoneString);
+                    ? peronSelected.getPhone().getPhoneNo() : "--";
+            out.printf((PRINT_LIST_FORMAT) + "%n", i + 1, truncatedName, idString, phoneString);
             printSingleDivider();
         }
     }
 
     private void printFindList(Person person) {
         assert person != null : "Only prints Person";
+        printDivider();
         out.printf((PRINT_LISTALL_FORMAT) + "%n", " ", "Name", "Id", "Phone", "Checked In");
         printDivider();
         String name = person.getName().getNameString();
+        String truncatedName = nameTruncator(name);
         String idString = person.getId().getIdString();
         String phoneString = (person.getPhone().isAvailable())
-                ? "--" : person.getPhone().getPhoneNo();
+                ? person.getPhone().getPhoneNo() : "--";
         String statusString = (person.getCheckedIn()) ? "Yes" : "No";
-        out.printf((PRINT_LISTALL_FORMAT) + "%n", 1, name, idString, phoneString, statusString);
-        printSingleDivider();
+        out.printf((PRINT_LISTALL_FORMAT) + "%n", 1, truncatedName, idString, phoneString, statusString);
+        printDivider();
 
     }
 
     private void printPersonsInListWithStatus(List<Person> persons) {
         for (int i = 0; i < persons.size(); ++i) {
             Person personSelected = persons.get(i);
             String name = personSelected.getName().getNameString();
+            String truncatedName = nameTruncator(name);
             String idString = personSelected.getId().getIdString();
             String phoneString = (personSelected.getPhone().isAvailable())
-                    ? "--" : personSelected.getPhone().getPhoneNo();
+                    ? personSelected.getPhone().getPhoneNo() : "--";
```

Comment:
since you use this a lot, perhaps you can abstract out to a method and use if condition in that method for readability.